zappa/__init__.py

```diff
@@ -1,35 +1,27 @@
+import os
 import sys
 from pathlib import Path
 
 
+# Alternative solution to: https://github.com/zappa/Zappa/issues/1140
 def running_in_docker() -> bool:
     """
     Determine if zappa is running in docker.
     - When docker is used allow usage of any python version
     """
-    # https://stackoverflow.com/a/20012536/24718
-    cgroup_content = Path("/proc/1/cgroup").read_text()
-    in_docker = "/docker/" in cgroup_content or "/lxc/" in cgroup_content
-    return in_docker
+    # https://stackoverflow.com/questions/63116419
+    running_in_docker_flag = os.getenv("ZAPPA_RUNNING_IN_DOCKER", "False").lower() in ("true", "1", "t")
```

Comment:
distutils has a strtobool function, it is deprecitated, but has slightly more expanded options.
https://github.com/python/cpython/blob/3.10/Lib/distutils/util.py#L308

```
in_docker = os.getenv("ZAPPA_RUNNING_IN_DOCKER", "False").lower() in ('y', 'yes', 't', 'true', 'on', '1')
```

Reply:
Good point, thanks for this